### PR TITLE
[DELIVERABLE 4] VnV Plan: All Nivetha's Sections

### LINKED ISSUE #175: [VnV Plan] Plan: Verification and Validation Team

### PR DESCRIPTION
**Pull request by nivethakuruparan**
## Description

Completed sections 3.1, 3.4, and 3.5 in the VnV Plan. Should close #175 #178 #180 

## How Has This Been Tested?

Compiled locally

### CODE REVIEW

File: docs/VnVPlan/VnVPlan.tex [hunk 1]
```diff
@@ -264,25 +271,45 @@ \subsection{Design Verification Plan}
...
+    \item \textbf{Unit Testing}: A comprehensive suite of unit tests will be established to validate the functionality of individual components within the optimizer. These tests will specifically focus on the effectiveness of the code refactoring methods employed by the optimizer, utilizing \texttt{pytest} for writing and executing these tests.
+    
+    \item \textbf{Static Code Analysis}: To maintain high code quality, static analysis tools such as \texttt{Pylint} and \texttt{Flake8} will be employed. These tools will help identify potential bugs, security vulnerabilities, and adherence to coding standards in the Python codebase, ensuring that the optimizer is both efficient and secure.
+    
+    \item \textbf{Code Walkthroughs and Reviews}: The development team will hold regular code reviews and walkthrough sessions to collaboratively evaluate the implementation of the source code optimizer. These sessions will focus on code quality, readability, and compliance with the project’s design patterns. Additionally, the final presentation in CAS 741 will provide an opportunity for a thorough code walkthrough, allowing peers to contribute feedback on usability and functionality.
```
[the remark this answers is not in the document]
Comment (nivethakuruparan) [reply docs/VnVPlan/VnVPlan.tex:307]: Oops that was what the blue comment said. What class are we?

File: docs/VnVPlan/VnVPlan.tex [hunk 2]
```diff
@@ -133,9 +133,16 @@ \section{Plan}
 
 \subsection{Verification and Validation Team}
 
-\wss{Your teammates.  Maybe your supervisor.
-  You should do more than list names.  You should say what each person's role is
-  for the project's verification.  A table is a good way to summarize this information.}
+The Verification and Validation (VnV) Team for the Source Code Optimizer project consists of the following members and their specific roles:
+
+\begin{itemize}
+    \item \textbf{Sevhena Walker}: Lead Tester. Oversees and coordinates the testing process, ensuring all feedback is applied and all project goals are met.
```
[the remark this answers is not in the document]
Comment (nivethakuruparan) [reply docs/VnVPlan/VnVPlan.tex:139]: I just pulled these out of thin air

File: docs/VnVPlan/VnVPlan.tex (hunk 1 above)
[the remark this answers is not in the document]
Comment (Ayushi1972) [reply docs/VnVPlan/VnVPlan.tex:307]: I think u can just remove it and keep "final presentation will..."

**Review by Ayushi1972 — APPROVED** [review]

**Review by Ayushi1972 — APPROVED** [review]

File: docs/VnVPlan/VnVPlan.tex [hunk 3]
```diff
@@ -272,25 +279,45 @@ \subsection{Design Verification Plan}
...
+    \item \textbf{Unit Testing}: A comprehensive suite of unit tests will be established to validate the functionality of individual components within the optimizer. These tests will specifically focus on the effectiveness of the code refactoring methods employed by the optimizer, utilizing \texttt{pytest} for writing and executing these tests.
+    
+    \item \textbf{Static Code Analysis}: To maintain high code quality, static analysis tools such as \texttt{Pylint} and \texttt{Flake8} will be employed. These tools will help identify potential bugs, security vulnerabilities, and adherence to coding standards in the Python codebase, ensuring that the optimizer is both efficient and secure.
+    
+    \item \textbf{Code Walkthroughs and Reviews}: The development team will hold regular code reviews and walkthrough sessions to collaboratively evaluate the implementation of the source code optimizer. These sessions will focus on code quality, readability, and compliance with the project’s design patterns. Additionally, the final presentation in CAS 741 will provide an opportunity for a thorough code walkthrough, allowing peers to contribute feedback on usability and functionality.
```
Comment (Sevhena) [inline docs/VnVPlan/VnVPlan.tex:315]:
```suggestion
    \item \textbf{Code Walkthroughs and Reviews}: The development team will hold regular code reviews and walkthrough sessions to collaboratively evaluate the implementation of the source code optimizer. These sessions will focus on code quality, readability, and compliance with the project’s design patterns. Additionally, the final presentation will provide an opportunity for a thorough code walkthrough, allowing peers to contribute feedback on usability and functionality.
```

**Review by Sevhena — CHANGES_REQUESTED** [review]

**Review by Sevhena — APPROVED** [review]
LGTM

**Pull request merged**